roles/linux-common/tasks/main.yml

```diff
@@ -47,6 +47,9 @@
     enabled: yes
     gpgcheck: no
 
+- name: Enable optional repos
+  command: yum-config-manager --enable rhui-REGION-rhel-server-extras rhui-REGION-rhel-server-optional
```

Comment:
Isn't this going to break on installs on VMWare using CentOS?

Reply:
I haven't tested but good catch. it may fail on CentOS. I did put a condition for that task.